src/lib/web-component/log-view/log-view.css

```diff
@@ -19,7 +24,11 @@
 }
 
 .log-details {
-  margin: 50px 15px 10px;
+  margin: 40px 0 10px;
+  white-space: pre-wrap;
+  max-height: 450px;
+  overflow-y: scroll;
+  padding-left: 20px;
```

Comment:
Indenting the layout a bit from left so that log string doesn't align too closely with the edge.